### PR TITLE
Bugfix: oc describe scc <name> printing error

### PR DESCRIPTION
**Pull request by byteor**
Describer printing bool pointer directly, it should print out bool variable.

Signed-off-by: Zhou Peng <p@ctriple.cn>

### CODE REVIEW

File: pkg/oc/lib/describe/describer.go [hunk 1]
```diff
@@ -1909,7 +1909,7 @@ func describeSecurityContextConstraints(scc *securityapi.SecurityContextConstrai
 
 		fmt.Fprintf(out, "Settings:\t\n")
 		fmt.Fprintf(out, "  Allow Privileged:\t%t\n", scc.AllowPrivilegedContainer)
-		fmt.Fprintf(out, "  Allow Privilege Escalation:\t%v\n", scc.AllowPrivilegeEscalation)
+		fmt.Fprintf(out, "  Allow Privilege Escalation:\t%t\n", *scc.AllowPrivilegeEscalation)
```
Comment (soltysh) [inline pkg/oc/lib/describe/describer.go]: You need a nil check before doing it this way.

File: pkg/oc/lib/describe/describer.go [hunk 2]
```diff
@@ -1909,7 +1909,13 @@ func describeSecurityContextConstraints(scc *securityapi.SecurityContextConstrai
 
 		fmt.Fprintf(out, "Settings:\t\n")
 		fmt.Fprintf(out, "  Allow Privileged:\t%t\n", scc.AllowPrivilegedContainer)
-		fmt.Fprintf(out, "  Allow Privilege Escalation:\t%v\n", scc.AllowPrivilegeEscalation)
+
+		allowPrivilegeEscalation := ""
```
Comment (soltysh) [inline pkg/oc/lib/describe/describer.go]: `<none>` should be used when not set, like in other cases.

**Comment by byteor** [comment]
/assign soltysh
please trigger test to merge

**Pull request merged**